src/plugins/telemetry_collection_manager/common/index.ts

```diff
@@ -8,3 +8,9 @@
 
 export const PLUGIN_ID = 'telemetryCollectionManager';
 export const PLUGIN_NAME = 'telemetry_collection_manager';
+
+/**
+ * The duration, in milliseconds, to cache stats
+ * Currently 4 hours.
```

Comment:
not: it's easy to forget to update the comment in the future. I'd remove it and use code instead:
```js
const hour = 1000 * 60 * 60;
export const CACHE_DURATION_MS = 4 * hour;
```